Tools/DumpRenderTree/TestRunner.cpp

```diff
@@ -1785,6 +1785,16 @@ static JSValueRef forceImmediateCompletionCallback(JSContextRef context, JSObjec
     return JSValueMakeUndefined(context);
 }
 
+static JSValueRef setTopContentInsetCallback(JSContextRef context, JSObjectRef function, JSObjectRef thisObject, size_t argumentCount, const JSValueRef arguments[], JSValueRef* exception)
+{
+    if (argumentCount != 1)
+        return JSValueMakeUndefined(context);
+    TestRunner* controller = static_cast<TestRunner*>(JSObjectGetPrivate(thisObject));
+    double contentInset = JSValueToNumber(context, arguments[0], exception);
+    controller->setTopContentInset(contentInset);
+    return JSValueMakeUndefined(context);
```

Comment:
We may want to return JSObjectMakeDeferredPromise and resolve it immediately in this case so the await does something in DRT.  I probably should've done that with flushConsoleLogs too.

Reply:
I can do that for both in another pr